Makefile

```diff
@@ -53,6 +63,9 @@ clean: ## Make a clean source tree
 lint: ## Check code style
 	pre-commit run --all-files
 
+nuke: ## Make clean + remove conda env
```

Comment:
This target name has kind of always bothered me.  Would you mind renaming this to `remove-env`?